packages/README.md

```diff
@@ -0,0 +1,11 @@
+# Elementor packages
+
+### Open Issues
```

Comment:
also need to decide how to install a package as a dependency of another package in the workspace. 
Currently, we need to `npm i` it and then change the version to `*`. 
Maybe we can use:
```
npm i --exact @elementor/locations@*
```

Reply:
We need to create a document with a general workflow: 
- how to install a package
- how to run scripts (build, lint, etc.)
- how to use external packages
- when to set on peer deps, when to on deps when on root dev deps.

so I think it should be a task with general workflow